lib/import-export-visitor.js

```diff
@@ -738,9 +740,9 @@ function toModuleExport(specifierMap) {
 
     parts.push(
       exported,
-      ":()=>",
+      ":function(){return ",
```

Comment:
Consistency is good, but I suspect avoiding arrow functions for `module.export` is less important than for `module.importSync`.